scripts/submit-speed-tile-work-service.py

```diff
@@ -128,17 +128,29 @@ def submit_jobs(batch_client, env, week, bbox):
         containerOverrides={
           'memory': 8192,
           'vcpus': 2,
-          'command': ['/scripts/speed-tile-work.py', '--environment', 'Ref::environment', '--tile-level', 'Ref::tile_level', '--tile-index', 'Ref::tile_index', '--week', 'Ref::week', '--concurrency', '2', '--max-tile-level', max_level]
+          'command': ['/scripts/speed-tile-work.py', '--environment', 'Ref::environment', '--tile-level', 'Ref::tile_level', '--tile-index', 'Ref::tile_index', '--week', 'Ref::week', '--concurrency', '2', '--max-tile-level', 'Ref::max_level', '--osmlr-version', 'Ref:osmlr_version']
         }
       )
       parent_id = submitted['jobId']
       logger.info('Job %s was submitted and got id %s' % (job_name, parent_id))
 
+def get_osmlr_version(version):
```

Comment:
here we check if they passed us a version and if so we check if its valid, if not we get the latest version